src/StringHelper.php

```diff
@@ -134,6 +134,35 @@ public static function truncateBegin(string $string, int $length, string $suffix
         return substr_replace($string, $suffix, 0, $length);
     }
 
+    /**
+     * Truncates a string in the middle. Keeping start and end.
+     * `StringHelper::truncateMiddle('Hello world number 2', 8)` produces "Hell...er 2".
+     *
+     * This method does not support html. It will strip all tags even if length is smaller.
+     *
+     * @param string $string    The string to truncate.
+     * @param int $length       How many characters from original string to include into truncated string.
+     * @param string $separator String to append in the middle of truncated string.
+     * @param string $encoding  The charset to use, defaults to charset currently used by application.
+     * @return string the truncated string.
+     */
+    public static function truncateMiddle($string, $length, $separator = '...', $encoding = 'UTF-8')
+    {
+        $string = strip_tags($string);
```

Comment:
Is it really needed?

Reply:
If they consider that it is not necessary, I can eliminate it.

Reply:
It is stated in phpdoc that it does not work with HTML so HTML is not expected in input. I'd remove it.

Reply:
Ok, no problem, updated :)